nonnie026/calculator
# ci: setup commitlint, husky, lint-staged, prettier and eslint

## Problem I Am Trying to Solve
- Setup vscode environment rules for linting and commiting

## Work Done
- Installed and set up husky, commitlint to enforce commit rules
- Install prettier and eslint with other plugins to enforce linting rules

## How to test
- Make a small change and attempt commiting it. See that husky and commitlint control this process

## Checklist (Acceptance Criteria)
- [x] Auto format *.js files on commit
- [x] Commits must obey commitlint rules
- [x] Project contains eslintrc and prettierrc files with correct rules
